client/store/actions.js

```diff
@@ -15,94 +15,90 @@ function clearStore({ commit }) {
 }
 
 function thingyTileAdd({ commit, state }, quantifiedThingy) {
-  if(!state.oodlet.id) { return false; }
-  commit('thingyTileAdd', quantifiedThingy);
+  if (!state.oodlet.id) {
+    return;
+  }
```

Comment:
This and all other if statements could save a lot of vertical space if you make them one-liner, but I guess it's a thing of preference. Like you said it's also important to have a clear picture of something that changes your app flow.